### ISSUE TITLE
warnings fire in React Native Web for passing props not supposed by View, and callbacks dont fire

### ISSUE THREAD

**Issue by faceyspacey** [oblador/react-native-animatable#59]
for example:

`Warning: Unknown props `easing`, `iterationCount`, `onAnimationBegin` on <div> tag. Remove these props from the element.`

In addition, the `onAnimationBegin/End` callbacks don't fire.

Works nicely otherwise. I assume the callbacks don't fire for some reason having to do with passing  props that aren't supported by `View/Text/Etc`.

**Comment by faceyspacey** [comment]
Here's a PR to fix the propTypes:

https://github.com/oblador/react-native-animatable/pull/60


...as for the callbacks, i checked the code, and it's because they only apply to `animation` not `transition`. 

I'll make a PR for that too. Any tips for how you want it done?

**Comment by oblador** [comment]
Your PR is now merged, thanks :-)

**Comment by oblador** [comment]
Read this too quickly, the callbacks are unrelated to the error and _should_ fire. Can you set up a runnable repo with a minimal example that reproduces this and open a new issue?

**Comment by faceyspacey** [comment]
a PR won't do much. The callbacks are simply missing for *"transitions"*. `onAnimationEnd` should be in places like this:

https://github.com/oblador/react-native-animatable/blob/master/index.js#L454


So "animations" have them, but not "transitions." 


....I actually invested a few hours into working on adding them a week ago when I first looked into this, but then I decided to roll my own solution unfortunately. ...I'll share the code I come up with later.
